webhook.go

```diff
@@ -99,9 +99,14 @@ type DNSSrvRecord struct {
 type Webhook struct {
 	// Accept is the encoding type of outgoing events. The following encoding types are supported, otherwise
 	// a 406 response code is returned: application/octet-stream, application/json, application/jsonl, application/msgpack.
-	// Note: An `Accept` of application/octet-stream or application/json will result in a single response for batch sizes of 0 or 1
-	// and batch sizes greater than 1 will result in a multipart response. An `Accept` of application/jsonl or application/msgpack
-	// will always result in a single response with a list of batched events for any batch size.
+	/*
+		Note:
+			An `Accept` of application/octet-stream supports single message return with payload of WRP
+			An `Accept` of application/json supports single message return with json form of WRP
```

Comment:
sounds like batching will only be available for  `application/jsonl` and `application/msgpack`, that works 👍

Reply:
I spoke to John about this & he suggested this set of values for better clarity:
`application/wrp+json` - one json encoded wrp message
`application/wrp+msgpack` - one msgpack encoded wrp message
`application/wrp+octet-stream` - one message with the wrp payload in the http payload
`application/wrp+jsonl` - multiple jsonl encoded wrp messages
`application/wrp+msgpackl` - multiple msgpackl encoded wrp messages

Reply:
nice! I really like this idea